src/lib.rs

```diff
@@ -37,6 +37,9 @@ extern crate core;
 #[cfg(feature = "alloc")]
 extern crate alloc;
 
+#[cfg(test)]
+extern crate sha2;
```

Comment:
Where is this used? I don't see anything that needs it...

Reply:
Ah, only in a doctest, good catch. I've removed it in 73a5f47. (I originally had the doctest as an actual test so that I could print output.)